xdem/coreg/base.py

```diff
@@ -1677,6 +1678,8 @@ class OutSpecificDict(TypedDict, total=False):
 class InAffineDict(TypedDict, total=False):
     """Keys and types of inputs associated with affine methods."""
 
+    # Estimated initial shift
+    initial_shift: tuple[float, float, float] | None
```

Comment:
Do we need to allow `tuple[float, float]` here as well? (Or did I miss the conversion somewhere?)